crates/iroha_core/src/tx.rs

```diff
@@ -30,7 +29,7 @@ use crate::{
 /// `AcceptedTransaction` — a transaction accepted by Iroha peer.
 #[derive(Debug, Clone, PartialEq, Eq)]
 #[repr(transparent)]
-pub struct AcceptedTransaction(pub(super) SignedTransaction);
+pub struct AcceptedTransaction(SignedTransaction);
```

Comment:
Shall we take this opportunity to update the doc comment?

Should we treat _accept_ as implicitly meaning static analysis?
I've always felt that _accept_ is a somewhat ambiguous term, since even after passing static validation and becoming _accepted_, there's still room for it to be _rejected_ during dynamic validation.